src/components/Layout/Header.tsx

```diff
@@ -125,6 +102,7 @@ const Header: React.FC<HeaderProps> = ({ searchBar = true }) => {
       ]}
       sessionState={sessionState}
       logoHref="/docs"
+      location={location}
```

Comment:
Piping this in instructs the header menu [to close when it changes](https://github.com/ably/ably-ui/pull/703/commits/82fe143f5a953421fa37fab99c777f600f1d38ba).